game/script-topics.rpy

```diff
@@ -1928,6 +1928,95 @@ label talk_using_headphones_carefully:
     n "Thanks for hearing me out!{w=0.2} Ehehe."
     return
 
+# Natsuki discusses her trademark hairstyle with the player
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_natsukis_hairstyle",
+            unlocked=True,
+            prompt="Why do you style your hair like that?",
+            conditional=None,
+            category=["Fashion", "Natsuki"],
+            player_says=True,
+            affinity_range=(jn_affinity.RUINED, jn_affinity.LOVE),
```

Comment:
I'm not 100% certain, but isn't this the default already? I mean it's no big deal to have it clarified above the topic, just *technically unnecessary*. But it can stay, causes no issues.

Reply:
Yep, if you're referring to the `conditional` clause then you're absolutely right - I remember Multi bringing this up and removing empty conditional clauses from some other features; this one just slipped through the net.

Resolved in next commit!